tests/integration/snapshots/tests__integration__cfc__ir__actions_debug.snap

```diff
@@ -6,60 +6,46 @@ expression: output_file_content_without_headers
 
```

Comment:
`codegen_to_string`, which is primarily used for integration tests, does not include a lowering stage (there's no need), so some tests which previously had init functions generated (due to the lowering being done in the `TypeAnnotator` during the resolve-step) no longer have them.